### ISSUE TITLE
Handoff question

### ISSUE THREAD

**Issue by Spanu18** [SkyrilHD/Dell-Exx50-Hackintosh#44]
Hi everyone,

I have a dell e5550, everything works perfectly and I use it every day, really stable. But I have a question: does handoff work in the last release?

Thanks,
Alessio

**Comment by SkyrilHD** [comment]
For Intel cards, handoff does work but since Monterey I experienced some inconsistencies. So your mileage may vary. AirDrop does not work on all versions.

For broadcom cards, it should work fine like on real Macs.

**Comment by 2wenty2wo** [comment]
> For broadcom cards, it should work fine like on real Macs.

100% works. I used an adapter cable and a real MacBook Air wifi card I got off eBay. After stuffing around with trying to get Intel to work, using a real Apple card is the best solution.

**Comment by SkyrilHD** [comment]
Glad to hear that. Which card did you fit in btw?

Closing issue as it is now unrelated to EFI

**Comment by 2wenty2wo** [comment]
I used a BCM94360CS2 card from a 13” MacBook Air with a extension adapter cable that converts it to A + E. Wrapped the board up in kapton tape and I trimmed the internal plastic casing of the E5550 to fit the card in.

P.S Thank you for sharing the EFI. It works perfectly and is so stable. I use this computer as my daily driver and haven’t had any issues with it, especially after swapping out the wifi/Bluetooth card.
